Makefile

```diff
@@ -232,14 +232,15 @@ ifeq (,$(PYPI_PLATFORM))
 PYPI_PLATFORM=$(DEFAULT_PLATFORM)
 endif
 
-pypi-push-master: build-all pypicheck pypi-version-check pypi-platform-check
-	pip install --user --upgrade setuptools wheel twine
+pypi-push-master: build-all pypicheck pypi-platform-check
+	pip install --upgrade setuptools wheel twine
 	rm -rf dist
 	python setup.py sdist bdist_wheel --plat-name=$(PYPI_PLATFORM)
+	twine upload -u="$(PYPITEST_USERNAME)" -p="$(PYPITEST_PASSWORD)" --repository-url https://test.pypi.org/legacy/ dist/*
 
-pypi-push-release-candidate: releasecheck pypi-push-master
+pypi-push-release-candidate: releasecheck pypi-version-check pypi-push-master
```

Comment:
do you want this one to be to the test repo as well?

Reply:
No this is the real push!